pkg/virt-controller/watch/node.go

```diff
@@ -274,7 +274,13 @@ func (c *NodeController) createAndApplyFailedVMINodeUnresponsivePatch(vmi *virtv
 	c.recorder.Event(vmi, v1.EventTypeNormal, NodeUnresponsiveReason, fmt.Sprintf("virt-handler on node %s is not responsive, marking VMI as failed", vmi.Status.NodeName))
 	logger.V(2).Infof("Moving vmi %s in namespace %s on unresponsive node to failed state", vmi.Name, vmi.Namespace)
 
-	patchBytes, err := generateFailedVMIPatch(vmi.Status.Reason)
+	patchSet := patch.New(patch.WithReplace("/status/phase", virtv1.Failed))
+	if vmi.Status.Reason == "" {
+		patchSet.AddOption(patch.WithAdd("/status/reason", NodeUnresponsiveReason))
+	} else {
+		patchSet.AddOption(patch.WithReplace("/status/reason", NodeUnresponsiveReason))
+	}
```

Comment:
Isn't add enough?

Reply:
@xpivarc I think so. The new code is replicated what the old one was doing, but let me change it

Reply:
Done